CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 #### Chore && Maintenance
 * Update example app to match Angular 8 Boilerplate ([#311](https://github.com/thymikee/jest-preset-angular/pull/309)).
+* Get rid of karma, extract jest config, fix paths ([#316](https://github.com/thymikee/jest-preset-angular/pull/316))
```

Comment:
Sorry for being nitpicking here, can you mention example app as well? Just to avoid confusion, as this is nothing about the preset itself.
Thanks!

Reply:
@wtho Would be this Ok: "Update example app: Get rid of karma, extract jest config, fix paths ([#316](https://github.com/thymikee/jest-preset-angular/pull/316))"?

Reply:
@wtho done